core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -299,6 +301,8 @@ object Boot extends IOApp with LazyLogging {
         metricsPrefix
       )
 
+      val workspaceManagerDAO = new HttpWorkspaceManagerDAO(conf.getString("workspaceManager.server"))
+
```

Comment:
the config changes need a corresponding PR in firecloud-develop, right?

Reply:
yes, i'll have a firecloud-develop PR for that in a bit